flocker_bb/builders/maint.py

```diff
@@ -25,6 +43,128 @@ def makeCleanOldBuildsFactory():
     return factory
 
 
+def get_creation_time(node):
+    """
+    Get the creation time of a libcloud node.
+
+    Rackspace and EC2 store the information in different metadta.
+
+    :return datetime:
+    """
+    date_string = node.extra.get("created", node.extra.get("launch_time"))
+    if date_string is None:
+        return None
```

Comment:
When would `launch_time` and `created` be missing?
And in this case it looks like we won't destroy the node.
Maybe document that situation and create a follow up issue.